src/main/java/hudson/plugins/ec2/ssh/EC2MacLauncher.java

```diff
@@ -151,6 +151,10 @@ protected void launchScript(EC2Computer computer, TaskListener listener) throws
         try {
             boolean isBootstrapped = bootstrap(computer, listener, template);
             if (isBootstrapped) {
+                logInfo(computer, listener, "SSH service responded. Waiting for service to stabilize");
```

Comment:
Can we add the number of seconds its sleeping here in the log? And only print these messages if bootdelay is > 0 else a 0 number sees these log lines spammed

Reply:
Makes sense, however in that case this should be done in all three places